File: gems/decomplex/examples/kotlin/function-lcom.kt
fun mixed(price: Int, tax: Int, logger: Logger): Result {
  val subtotal = price + tax
  val total = subtotal * 2
  val rounded = total.round()

  val timestamp = now()
  val buffer = Buffer.init()
  buffer.push(timestamp)
  logger.info(buffer)

  return Result.init(rounded, buffer)
}

File: gems/decomplex/examples/typescript/function-lcom.ts
function mixed(price: number, tax: number, logger: Logger) {
  const subtotal = price + tax;
  const total = subtotal * 2;
  const rounded = total.round();

  const timestamp = now();
  const buffer = Buffer.init();
  buffer.push(timestamp);
  logger.info(buffer);

  return Result.init(rounded, buffer);
}
